pretrain.py

```diff
@@ -6,6 +6,11 @@
 
 from autowebcompat import network, utils
 
+SAMPLE_SIZE = 3000
```

Comment:
While I like these changes, moving the constants at the top, they are unrelated to the objective of this PR, so they should be in a separate PR.